docs/rfc/luthier-design.md

```diff
@@ -240,18 +243,20 @@ define and use a custom key value store.
 
 Fili supports the following built in KeyValueStores:
 
-    * fili-map - Uses an in-memory data structure to handle point lookups. Not
-        recommended for large dimensions.
-        Additional keys:
-            None
+* fili-map - Uses an in-memory data structure to handle point lookups. Not
+    recommended for large dimensions.
+    
+    Additional keys:
+        None
+
+* fili-reddis - Uses a Redis cluster to perform point lookups.
 
-    * fili-reddis - Uses a Redis cluster to perform point lookups.
-        Additional keys:
-            redisNamespace - The reddis namespace to use
-            storeName - The reddis store to use
-            reddisConfiguration - A table with the following keys:
-                * host - The host running the Reddis service
-                * port - The port to contact the host through. Optional.
+    Additional keys:
+    * redisNamespace - The reddis namespace to use
+    * storeName - The reddis store to use
+    * reddisConfiguration - A table with the following keys:
+    * host - The host running the Reddis service
```

Comment:
`host` and `port` should use `**`. It's a sublist underneath `reddisConfiguration`

Reply:
I have overlooked this, thank you for pointing it out.